test/mocha/10-api.js

```diff
@@ -1,3 +1,745 @@
+/*!
+ * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const bedrock = require('bedrock');
+const {config} = bedrock;
+const helpers = require('./helpers');
+const mockData = require('./mock.data');
+const {httpClient} = require('@digitalbazaar/http-client');
+const brAccount = require('bedrock-account');
+const brAuthnToken = require('bedrock-authn-token');
+const {authenticator} = require('otplib');
```

Comment:
This is not required in this PR, but we might consider the async version of authenticator in the future:

https://github.com/yeojz/otplib/blob/master/packages/otplib-core-async/README.md

@mattcollier  @dlongley